mentors/prolog/a.json

```
@@ -1,10 +1,19 @@
 [
  {
-  "github_username": "Average-user",
-  "name": "Lucas Polymeris",
+  "github_username": "AnnieAtDataChemist",
+  "name": "Anne Ogborn",
   "link_text": null,
   "link_url": null,
   "avatar_url": null,
-  "bio": "I'd like to make Prolog a better known programming language."
- }
+  "bio": "Contributor to SWI-Prolog and all around Prolog enthusiast"
+ },
```

```suggestion
  },
```
Line 9 should be indented a total 2 spaces, just like line 10.